### PR TITLE
Try install in Appveyor (and docs as well in Travis)

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Comment by mattcaswell** [comment]
What is happening with this? Looks like Travis/Appveyor failed?

**Comment by mattcaswell** [comment]
Ping @levitte. What is the status of this?

**Comment by levitte** [comment]
I've just rebased to a fresher master, let's hope that makes things better.

**Comment by levitte** [comment]
This now includes the changes from #1238, which I think fixes the issue that got this PR to fail on Appveyor.

**Comment by levitte** [comment]
Hey, looks promising now :-)

**Comment by levitte** [comment]
Yes, this went  through.  @mattcaswell ?

File: test/testlib/OpenSSL/Test.pm [hunk 1]
```diff
@@ -845,10 +843,22 @@ sub __cwd {
...
+
+    if ($opts{cleanup}) {
+	rmtree(".", { safe => 0, keep_root => 1 });
+    }
+
+    %directories = %tmp_directories;
+    foreach (keys %tmp_ENV) {
+        $ENV{$_} = $tmp_ENV{$_};
+    }
+
```
Comment (mattcaswell) [inline test/testlib/OpenSSL/Test.pm]: The changes in this file look Ok - but I guess I don't understand the problem you described in the commit description. I can't work out what this solves.

Comment (levitte) [inline test/testlib/OpenSSL/Test.pm]: Say we just want to keep trick of where the source top is, so we'd have a `$directories{srctop} = ".."`, for example.

Now, consider we move into the subdirectory `foo`, and to do good, we want to make sure we know where the source top is relative to the new location.

The problem actually starts with `rel2abs`, which calculates the absolute directory for a given input.  Which result do you think would be true, doing `rel2abs($directories{srctop})` before or after doing `chdir($dir)`?

Comment (mattcaswell) [inline test/testlib/OpenSSL/Test.pm]: Ahhh...right.

**Comment by mattcaswell** [comment]
+1

**Comment by levitte** [comment]
Cool, merged

**Pull request closed without merging**